pkg/services/ngalert/notifier/legacy_storage/compat.go

```diff
@@ -15,3 +22,42 @@ func UidToName(uid string) (string, error) {
 	}
 	return string(data), nil
 }
+
+func IntegrationToPostableGrafanaReceiver(integration *models.Integration) (*apimodels.PostableGrafanaReceiver, error) {
+	postable := &apimodels.PostableGrafanaReceiver{
+		UID:                   integration.UID,
```

Comment:
Here or somewhere else upstream we should generate new GUID. I could not find any code in the new service that does this. 

Also, we need to make sure that this GUID is unique across the Alertmanager configuration